src/map/map.cpp

```diff
@@ -178,6 +178,12 @@ int32 do_init(int32 argc, char** argv)
         }
     }
 
+    if (!std::filesystem:exists("./scripts/settings/main.lua"))
+    {
+        ShowError("FAIL. See /scripts/settings/ReadMe.md immediately.");
```

Comment:
Markdown convention is for the file to be caps: `README. md`

Reply:
ironically the other one is in full lowercase https://github.com/LandSandBoat/server/blob/base/conf/readme.md